inference-engine/tests/functional/shared_test_classes/src/subgraph/memory_concat.cpp

```diff
@@ -0,0 +1,124 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
the model in this test is more complicated than just memory layer with concat layer. It at least contains FQ and strided slice. I'd consider to rename the file to reflect this layers configuration, otherwise this test may conflict with simpler tests.

Reply:
renamed